<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keypad</class>
 <widget class="QWidget" name="keypad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color:blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>CLR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{image: url(:/new/prefix1/Rectangle 18.png);}

QPushButton:hover {
image: url(:/new/prefix1/Rectangle 18.png);


    /* Style when hovered over */
    background-color: blue;
    color: black;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
